recipes-kernel/kernel-modules/kernel-module-imx-gpu-viv_6.4.0.p2.4.bb

```diff
@@ -7,14 +7,14 @@ allowing flexibility to use a newer graphics release with an older kernel."
 LICENSE = "GPLv2"
 LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/GPL-2.0;md5=801f80980d171dd6425610833a22dbe6"
 
-SRCBRANCH = "lf-5.4.y"
+SRCBRANCH = "imx_5.4.24_2.1.0"
 LOCALVERSION = "-${SRCBRANCH}"
 KERNEL_SRC ?= "git://source.codeaurora.org/external/imx/linux-imx.git;protocol=https"
 SRC_URI = " \
     ${KERNEL_SRC};branch=${SRCBRANCH};subpath=drivers/mxc/gpu-viv;destsuffix=git/src \
     file://Add-makefile.patch \
 "
-SRCREV = "f8118585ee3c7025265b28985fdfe0af96a84466"
+SRCREV = "2f6f3e92a82e51fe39b53ee758273bef26964836"
```

Comment:
@gibsson can you handle our kernel module fork?

Reply:
@otavio yes will do!

Reply:
@otavio: I split it in 2 patches: https://github.com/Freescale/meta-freescale/pull/462
The first one, update to 6.4.0.p2.0 should be backported to dunfell whereas the second one should only be on master.
Hope it makes sense.